roles/openshift_excluder/tasks/status.yml

```diff
@@ -17,37 +17,37 @@
   register: docker_excluder_installed
   failed_when: false
 
-- name: Update to latest excluder packages
-  package:
-    name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
-  when:
-  - "{{ openshift_excluder_installed.installed_versions | default([]) | length > 0 }}"
-  - not openshift.common.is_containerized | bool
-
-- name: Update to the latest docker-excluder packages
-  package:
-    name: "{{ openshift.common.service_type }}-docker-excluder"
-    state: latest
-  when:
-  - "{{ docker_excluder_installed.installed_versions | default([]) | length > 0 }}"
-  - not openshift.common.is_containerized | bool
+- block:
+  - name: Update excluder packages if requested
```

Comment:
This currently does not make sense. During the installation if the excludes are not installed, the task is skipped. If the excluders are installed, the are not update. IIRC, @detiber's idea was not to upgrade excludes to 3.5 if we are deploying 3.4 openshift. So either we can check what OS version is installed and based on that decide which excluders version we want to update to.

### Some scenarios
Assume ``3.4.1.7-1`` excluders are installed and other ``3.4.1.8-1`` atomic-openshift rpms are installed as well. Should we update excluders to ``3.4.1.8-1``? Or is that something a user should take care of? I would say we should take care of that if we are re-running installation part of the ansible

Assume ``3.4.1.8-1`` exluders and other atomic-openshifts rpms are installed. At the same time 3.5 repo is enabled. However, ``openshift_release=v3.4``. We should not update excluders to the latest when running the installation again (or for the first time).

Assume ``3.3.1.8-1`` excluders and other atomic-openshifts rpms are installed. At the same time 3.4 and 3.5 repos are enabled. User is running upgrade from 3.3 to 3.4 with ``openshift_release=v3.4``. We should not upgrade excluders to ``3.5.*``. Instead of that, we should update excluders to the latest 3.4.